lndmanage/settings.py

```diff
@@ -36,19 +38,26 @@ def parse_env(key, default, _type=str):
 # of the to be balanced channels
 REBALANCING_TRIALS = parse_env('LNDMANAGE_REBALANCING_TRIALS', '10', int)
 
+# ======== LNDMANAGED =========
+# -------- channel acceptor --------
+CHACC_MIN_CHANNEL_SIZE_PRIVATE = parse_env('LNDMANAGED_CHACC_MIN_CHANNEL_SIZE_PRIVATE', '0', int)
+CHACC_MAX_CHANNEL_SIZE_PRIVATE = parse_env('LNDMANAGED_CHACC_MAX_CHANNEL_SIZE_PRIVATE', '16777215', int)
+CHACC_MIN_CHANNEL_SIZE_PUBLIC = parse_env('LNDMANAGED_CHACC_MIN_CHANNEL_SIZE_PUBLIC', '0', int)
+CHACC_MAX_CHANNEL_SIZE_PUBLIC = parse_env('LNDMANAGED_CHACC_MAX_CHANNEL_SIZE_PUBLIC', '16777215', int)
 
-logger_config = None
+# define some globals for logging
+lndmanage_log_config = None
+lndmanaged_log_config = None
 home_dir = None
 
 
 def set_lndmanage_home_dir(directory=None):
-    """
-    Sets the correct path to the lndmanage home folder.
+    """Sets the correct path to the lndmanage home folder.
 
     :param directory: home folder, overwrites default
     :type directory: str
     """
-    global home_dir, logger_config
+    global home_dir, lndmanage_log_config, lndmanaged_log_config
```

Comment:
How would you feel about a way to get home_dir/log paths through a way that does not require `global` and permuting global state and side-effects through import ordering?

I get that maybe that's not of super-high prio but as seen in #125 this already has unexpected behavior and maybe rather than going further in this direction, it could be nice to identify a more predictable alternative?

What I have in mind would be to remove the `set_lndmanage_home_dir` function (do you ever really *want* to change it during the process lifetime?) and instead use pure getters/accessors here.

For things like tests, `os.environ.setdefault` should work fine, though maybe there are other/better options for that

Reply:
Agree, the settings file is kind of annoying. I'll have a look when dealing with the `lndmanaged` configuration, which will be the next step.